### PR TITLE
Update macaw to 0.3.0

### PR DESCRIPTION
**Pull request by dominik-zeglen**
I want to merge this change because it updates macaw to 0.3.0 and uses its components - `Button`, `IconButton` and `Pagination`.

**PR intended to be tested with API branch:** 3.0

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://master.staging.saleor.cloud/graphql/

### CODE REVIEW

File: src/apps/components/CustomAppDetailsPage/CustomAppDetailsPage.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import { Button } from "@material-ui/core";
+import "@material-ui/core";
```
Comment (jwm0) [inline src/apps/components/CustomAppDetailsPage/CustomAppDetailsPage.tsx:1]: leftover

File: src/apps/components/CustomAppTokens/CustomAppTokens.tsx [hunk 2]
```diff
@@ -80,6 +79,7 @@ const CustomAppTokens: React.FC<CustomAppTokensProps> = props => {
                 </TableCell>
                 <TableCell className={classes.colActions}>
                   <IconButton
+                    variant="secondary"
                     color="primary"
```
Comment (jwm0) [inline src/apps/components/CustomAppTokens/CustomAppTokens.tsx:83]: What's the default color for `IconButton`? If it's also "primary" then we should skip it like we do in Buttons

File: src/attributes/views/AttributeList/AttributeList.tsx [hunk 3]
```diff
@@ -1,4 +1,5 @@
-import { IconButton } from "@material-ui/core";
+import "@material-ui/core";
```
Comment (jwm0) [inline src/attributes/views/AttributeList/AttributeList.tsx:1]: leftover

File: src/components/AssignAttributeDialog/AssignAttributeDialog.tsx [hunk 4]
```diff
@@ -217,7 +216,7 @@ const AssignAttributeDialog: React.FC<AssignAttributeDialogProps> = ({
         <ConfirmButton
           transitionState={confirmButtonState}
           color="primary"
```
Comment (jwm0) [inline src/components/AssignAttributeDialog/AssignAttributeDialog.tsx:218]: Same as with IconButton, shouldn't we default to `primary` in all instances?

File: src/components/AssignAttributeDialog/AssignAttributeDialog.tsx [hunk 5]
```diff
@@ -25,7 +24,7 @@ import useModalDialogErrors from "@saleor/hooks/useModalDialogErrors";
 import useModalDialogOpen from "@saleor/hooks/useModalDialogOpen";
 import useSearchQuery from "@saleor/hooks/useSearchQuery";
 import { buttonMessages } from "@saleor/intl";
-import { makeStyles } from "@saleor/macaw-ui";
+import { Button, makeStyles } from "@saleor/macaw-ui";
```
Comment (jwm0) [inline src/components/AssignAttributeDialog/AssignAttributeDialog.tsx:27]: We're changing the import but the color/variant remains the same. Is the new default <Button /> equivalent to the old one?

File: src/components/ColumnPicker/ColumnPickerButton.tsx [hunk 6]
```diff
@@ -1,7 +1,8 @@
-import { Button } from "@material-ui/core";
+import "@material-ui/core";
```
Comment (jwm0) [inline src/components/ColumnPicker/ColumnPickerButton.tsx:1]: leftover

File: src/components/DeleteButton/DeleteButton.tsx [hunk 7]
```diff
@@ -1,6 +1,7 @@
-import { Button } from "@material-ui/core";
+import "@material-ui/core";
```
Comment (jwm0) [inline src/components/DeleteButton/DeleteButton.tsx:1]: leftover

File: src/components/FilterBar/FilterBar.tsx [hunk 8]
```diff
@@ -1,5 +1,6 @@
-import { Button } from "@material-ui/core";
-import { makeStyles } from "@saleor/macaw-ui";
+import "@material-ui/core";
```
Comment (jwm0) [inline src/components/FilterBar/FilterBar.tsx:1]: leftover

File: src/components/SaveFilterTabDialog/SaveFilterTabDialog.tsx [hunk 9]
```diff
@@ -1,12 +1,12 @@
 import {
-  Button,
   Dialog,
   DialogActions,
   DialogContent,
   DialogTitle,
   TextField
 } from "@material-ui/core";
 import { buttonMessages } from "@saleor/intl";
+import { Button } from "@saleor/macaw-ui";
```
Comment (jwm0) [inline src/components/SaveFilterTabDialog/SaveFilterTabDialog.tsx:9]: Same as above, import changes but the component props remain the same - make sure it's intentional

File: src/components/TablePagination/TablePagination.tsx [hunk 10]
```diff
@@ -1,124 +1,59 @@
-import { TableCell, Toolbar } from "@material-ui/core";
-import { IconButtonProps } from "@material-ui/core/IconButton";
-import RowNumberSelect from "@saleor/components/RowNumberSelect";
-import { makeStyles } from "@saleor/macaw-ui";
-import { maybe } from "@saleor/misc";
+import { TableCell } from "@material-ui/core";
+import {
+  Pagination as MacawPagination,
+  PaginationProps as MacawPaginationProps
```
Comment (jwm0) [inline src/components/TablePagination/TablePagination.tsx:4]: What's the alias for?

File: src/components/TablePagination/TablePaginationActions.tsx [hunk 11]
```diff
@@ -1,8 +1,9 @@
-import { IconButton } from "@material-ui/core";
+import "@material-ui/core";
```
Comment (jwm0) [inline src/components/TablePagination/TablePaginationActions.tsx:1]: leftover

File: src/navigation/components/MenuListPage/MenuListPage.tsx [hunk 12]
```diff
@@ -1,8 +1,9 @@
-import { Button } from "@material-ui/core";
+import "@material-ui/core";
```
Comment (jwm0) [inline src/navigation/components/MenuListPage/MenuListPage.tsx:1]: cmd + f this guy

File: src/orders/components/OrderFulfillmentCancelDialog/OrderFulfillmentCancelDialog.tsx [hunk 13]
```diff
@@ -15,7 +14,7 @@ import SingleAutocompleteSelectField from "@saleor/components/SingleAutocomplete
 import { OrderErrorFragment } from "@saleor/fragments/types/OrderErrorFragment";
 import { WarehouseFragment } from "@saleor/fragments/types/WarehouseFragment";
 import { buttonMessages } from "@saleor/intl";
-import { makeStyles } from "@saleor/macaw-ui";
+import { Button, makeStyles } from "@saleor/macaw-ui";
```
Comment (jwm0) [inline src/orders/components/OrderFulfillmentCancelDialog/OrderFulfillmentCancelDialog.tsx:17]: Import changes, props remain the same - make sure it's intentional

**Review by jwm0 — CHANGES_REQUESTED** [review]

File: src/apps/components/CustomAppTokens/CustomAppTokens.tsx (hunk 2 above)
Comment (dominik-zeglen) [reply src/apps/components/CustomAppTokens/CustomAppTokens.tsx:83]: It's `default`, they become `primary` on hover

File: src/components/TablePagination/TablePagination.tsx (hunk 10 above)
Comment (dominik-zeglen) [reply src/components/TablePagination/TablePagination.tsx:4]: A leftover

File: src/orders/components/OrderFulfillmentCancelDialog/OrderFulfillmentCancelDialog.tsx (hunk 13 above)
Comment (dominik-zeglen) [reply src/orders/components/OrderFulfillmentCancelDialog/OrderFulfillmentCancelDialog.tsx:17]: It is, we should be using only Macaw's buttons

File: src/components/AssignAttributeDialog/AssignAttributeDialog.tsx (hunk 4 above)
Comment (dominik-zeglen) [reply src/components/AssignAttributeDialog/AssignAttributeDialog.tsx:218]: We should 🤔

File: src/components/AssignAttributeDialog/AssignAttributeDialog.tsx (hunk 5 above)
Comment (dominik-zeglen) [reply src/components/AssignAttributeDialog/AssignAttributeDialog.tsx:27]: It is

**Review by Droniu — APPROVED** [review]
Since `tertiary` variant is default for buttons it can be omitted sometimes, but it doesn't affect readability. Overall looks good

**Review by jwm0 — APPROVED** [review]

**Pull request merged**